frontend/app/.env

```diff
@@ -2,4 +2,5 @@ VITE_BACKEND_URL=http://127.0.0.1:4242
 VITE_COLIBRI_URL=http://127.0.0.1:4343
 VITE_I18N_LOCALE=en
 VITE_I18N_FALLBACK_LOCALE=en
+VITE_WALLET_CONNECT_PROJECT_ID=eca1bd3842e084bfc8ef309b6eb584db
```

Comment:
For the project_id, wallet connect do some endpoint call, that we can see from the network tab, and the project id is exposed anyway, so I think it should be fine (?)

Otherwise, we can let user put their own projectId, but it needs more work